src/pages/ContactForm/DropdownComponent.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+  import { StyleSheet } from 'react-native';
+  import { Dropdown } from 'react-native-element-dropdown';
+  import AntDesign from '@expo/vector-icons/AntDesign';
+
+  const data = [
+    { label: 'Item 1', value: '1' },
+    { label: 'Item 2', value: '2' },
+    { label: 'Item 3', value: '3' },
+  ];
+
+  const DropdownComponent = () => {
+    const [value, setValue] = useState(null);
+
+    return (
+      <Dropdown
+        className="text-[16px] pt-2 pb-4 bg-gray-100 text-center"
+        placeholderTextColor="gray"
+        selectedTextStyle={styles.selectedTextStyle}
+        inputSearchStyle={styles.inputSearchStyle}
+        iconStyle={styles.iconStyle}
+        data={data}
+        search
+        labelField="label"
+        valueField="value"
+        placeholder="..."
```

Comment:
To render placeholder's text by any condition, we have talked about creating isFocused state and update when onFocus or onBlur event triggered at Dropdown component